tests/playwright/Report.md

```diff
@@ -0,0 +1,213 @@
+
+The above chart further shows that Playwright is much faster than WebdriverIO in terms of speed.
+
+#### Why adapt Playwright
+
+Playwright offers some great reasons such as;
+
+* It's simple to set up
+* Stable features.
+* Ability to install Chrome, Firefox or WebKit (Safari) automatically
+* Bidirectional (events) – automating things like console logs is easy
+* Maintained by [Microsoft people](https://blog.logrocket.com/playwright-vs-puppeteer/) with experience maintaining Puppeteer
+
+However, Playwright also presents some cons such as;
+
+* It is very new so the APIs are evolving
+* Has no support for IE11 or non-browser platforms
+* Documentations and community are not as good as the other framework yet.
+
+#### Advantages and disadvantages of Migrating from WebdriverIO
+
+From the various evaluations done against WebdriverIO, there seems to be great potential replacement in the future and Puppeteer and Playwright seem to stand out in terms of both speed, Stability. However, the question being asked is, is it worth migrating from WebdriverIO to another test automation framework? Let’s look at the Advantages and disadvantages of a potential migration;
+
+##### Advantages
+
+* WebdriverIO is [JavaScript](https://en.wikipedia.org/wiki/JavaScript) based and is built over [Node.js](https://en.wikipedia.org/wiki/Node.js) just like Puppeteer and Playwright thus it will be easier for developers since there would be no differences in programming language.
+
+##### Disadvantages
+
+* Investment in WebdriverIO tool. Most tests are already written in WebdriverIO. About 30 MediaWiki repositories use it, so migrating from it to another tool would require a great deal of time and manpower.
+* Update of documentation. Most documentation is already written and has references to WebdriverIO. Updating the documentation would require some time.
+
+### What Next
+
+* Refactor the existing test
+* Evaluate Playwright against Puppeteer
+
+### Summary
+
+Statistics show that Playwright is a better alternative tool over WebdriverIO in terms of speed. However, WebdriverIO seems to be a great tool given that it keeps evolving, new and modern features are being added to it and the community is still vibrant and supportive. As of the time of the evaluation, there are no plans to switch to a different test automation framework but this could change in the future.
+
+### Acknowledgement
+
+The completion of this project wouldn't have been a success without the support of my mentors, [Željko Filipin](https://filipin.eu/), [Vidhi Mody](https://www.vidhimody.me/) and [Soham Parekh](https://www.sohamp.dev/about/). I do appreciate the constant guidance, meetings, advice, code reviews, challenging, pair programmings, to mention but a few, that were offered during the project execution.
```

Comment:
My guess is that @und3fined-v01d would prefer link to his site's home page, but I'll leave that to him. 😄 

```suggestion
The completion of this project wouldn't have been a success without the support of my mentors, [Željko Filipin](https://filipin.eu/), [Vidhi Mody](https://www.vidhimody.me/) and [Soham Parekh](https://www.sohamp.dev/). I do appreciate the constant guidance, meetings, advice, code reviews, challenging, pair programmings, to mention but a few, that were offered during the project execution.
```